lamar/utils/localization.py

```diff
@@ -1,9 +1,12 @@
 from pathlib import Path
-from typing import List, Tuple, Callable
-from collections import defaultdict
+from typing import Any, Dict, List, Tuple, Callable
 import numpy as np
 
 import pycolmap
+try:
+    import poselib
```

Comment:
Maybe adding the installation instructions in README, perhaps also updating the pyproject and Dockerfile. Same for LightGlue.